### DISCUSSION TITLE
Validate my Unraid/Docker Compose setup

### DISCUSSION THREAD

**Discussion by sohail579** [immich-app/immich#2057] [Q&A]
With this latest version and the need to add another container I have decided to redo my docker compose file and env file.

firstly i have noticed that we now have a volume in the machine learning container which i did not have originally this is the /cache.  I have mapped this to my app data as  i have done with my database.

 immich-machine-learning:
    container_name: immich_machine_learning
    image: ghcr.io/immich-app/immich-machine-learning:release
    volumes:
      - ${UPLOAD_LOCATION}:/usr/src/app/upload
      **- /mnt/user/appdata/immich/mlcache:/cache**
    env_file:
      - .env
    environment:
      - NODE_ENV=production
    restart: always

is this correct? or does it just need to go to /tmp and run on memory?

I have also done the same to the typesense data which is now mapped to my cache drive

  typesense:
    container_name: immich_typesense
    image: typesense/typesense:0.24.0
    environment:
      - TYPESENSE_API_KEY=${TYPESENSE_API_KEY}
      - TYPESENSE_DATA_DIR=/data
    logging:
      driver: none
    volumes:
      **- /mnt/user/appdata/immich/tsdata:/data**
    restart: always

in the .env file there is no longer a need for the JWT SECRET will this cause me issues with my existing users?

**Comment by bo0tzz** [comment]
You can compare your files to those in the latest release ([docker-compose.yml](https://github.com/immich-app/immich/releases/latest/download/docker-compose.yml), [.env](https://github.com/immich-app/immich/releases/latest/download/example.env)) to see which volumes, env vars, etc. you need.

JWT_SECRET was a removed a while ago, it doesn't do anything anymore so it's safe to remove.

**Reply by sohail579** [reply]
Thanks that is what i have done but just wated to make sure that the changes I have made to my volume mappings are correct?
Machine Learning Cache - /mnt/user/appdata/immich/mlcache:/cache  i have mapped this to a location on my Appdata..  is this ok? or just it just go to /tmp - Memory?
and secondly to ensure its ok to move my typesense data to the appdata folder too? /mnt/user/appdata/immich/tsdata:/data

**Reply by bo0tzz** [reply]
Should be ok, yes

**Reply by sohail579** [reply]
great thanks for confirming
